### PR TITLE
GraphQL file generation tests

### PR DESCRIPTION
**Pull request by abhishoya-gs**
Depends on https://github.com/finos/legend-engine/pull/1218

### CODE REVIEW

File: legend-sdlc-generation-file/pom.xml [hunk 1]
```diff
@@ -128,6 +128,12 @@
             <artifactId>legend-engine-external-format-jsonSchema</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.finos.legend.engine</groupId>
+            <artifactId>legend-engine-xt-graphQL-query</artifactId>
+            <scope>test</scope>
+        </dependency>
```
Comment (kevin-m-knight-gs) [inline legend-sdlc-generation-file/pom.xml:136]: Why is there a need to add this test dependency here? I don't see any tests or test data being updating in this module.

**Review by kevin-m-knight-gs — COMMENTED** [review]
I think the title of this PR is a little misleading. This doesn't enable GraphQL file generation in Studio pipelines. That is done elsewhere. This adds tests for GraphQL generation in legend-sdlc. That's great! But I think the PR title should be changed to reflect that.

Also, I had a question on one part of the change (see below).

**Comment by epsstan** [comment]
Looks like the generated .graphql outputs are not correct. In engine, didn't we change the behavior from a single file with all the GraphQL types vs a file per GraphQL type.

File: legend-sdlc-generation-file/pom.xml (hunk 1 above)
Comment (abhishoya-gs) [reply legend-sdlc-generation-file/pom.xml]: Have removed it

**Pull request closed without merging**